ZenPacks/zenoss/Docker/parsers/cgroupfs.py

```diff
@@ -178,9 +178,15 @@ class cgroupfs(CommandParser):
 
     createDefaultEventUsingExitCode = False
 
+    def dataForParser(self, context, dp):
+        return {
+            'cgroup_path': getattr(context, 'cgroup_path', '/sys/fs/cgroup'),
+        }
+
     def processResults(self, cmd, result):
         # Sort the output into a more accessible data structure.
-        data = self.dataFromLines(cmd.ds, cmd.component, cmd.result.output)
+        cgroup_path = cmd.points[0].data['cgroup_path']
```

Comment:
This can result in a KeyError using a cached config from the previous Docker ZenPack version. It should be the following instead.
```
cgroup_path = cmd.points[0].data.get('cgroup_path', '/sys/fs/cgroup')
```